src/query/vacuum.c

```diff
@@ -2847,9 +2847,9 @@ vacuum_process_vacuum_data (THREAD_ENTRY * thread_p)
   vacuum_finalize (thread_p);
 
   vacuum_Data.is_vacuum_complete = true;
-
-  PERF_UTIME_TRACKER_TIME (thread_p, &perf_tracker, mnt_vac_master_time);
 #endif /* SA_MODE */
+
+  PERF_UTIME_TRACKER_TIME (thread_p, &perf_tracker, PSTAT_VAC_MASTER);
```

Comment:
I consulted with Remzi and it should be applied to both cases. I removed a
redundant call that was made
in if (!lf_circular_queue_produce (vacuum_Job_queue, &vacuum_job_entry)).

       Antonio

On Tue, Aug 23, 2016 at 10:17 AM, ESeok Oh <notifications@github.com> wrote:

> In src/query/vacuum.c
> <https://github.com/CUBRID/cubrid/pull/176#discussion_r75813092>:
>
> >  #endif /* SA_MODE */
> > +
> > +  PERF_UTIME_TRACKER_TIME (thread_p, &perf_tracker, PSTAT_VAC_MASTER);
>
> @asturzu <https://github.com/asturzu> @ribram <https://github.com/ribram>
>
> I'm not sure this should also be applied to SERVER_MODE. I think the
> existing code is correct.
> Would you confirm it again?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/CUBRID/cubrid/pull/176/files/bfa127cc2a731fac8c9290f7817e6b380acf5f4d#r75813092>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AQVQwrqM1by3-cZjA0w8umulZKOTqFhmks5qip5wgaJpZM4JYBXn>
> .
>